### PR TITLE
WEBAPP-72, WEBAPP-58, WEBAPP-65, WEBAPP-34, WEBAPP-81

### PR DESCRIPTION
**Pull request by maxammann**
Fixes:

- Scroll behaviour
- Remove sitemap
- Translations
- Location name
- Add version display

### CODE REVIEW

File: tools/webpack.config.js [hunk 1]
```diff
@@ -228,6 +228,8 @@ const config = {
 // Optimize the bundle in release (production) mode
 if (!isDebug) {
   config.plugins.push(new webpack.optimize.UglifyJsPlugin({
+    parallel: true,
+    cache: true,
```
Comment (Schedulaar) [inline tools/webpack.config.js]: What do these two?

File: src/components/RichLayout/Footer.js [hunk 2]
```diff
@@ -26,10 +26,21 @@ class Footer extends React.Component {
     }
   }
 
+  getVersion () {
+    // eslint-disable-next-line no-undef
+    if (__DEV__) {
+      // eslint-disable-next-line no-undef
+      return <div className={style.item}>{__VERSION__}</div>
```
Comment (Schedulaar) [inline src/components/RichLayout/Footer.js]: __VERSION__ only includes a commit hash when the version is not exactly on that hash, right?

Comment (Schedulaar) [reply src/components/RichLayout/Footer.js]: There needs to be a little bit more visual separation of the disclaimer link and the version. Maybe you can use '|' for that?

Comment (maxammann) [reply src/components/RichLayout/Footer.js]: This is just for debugging so I think visuals are not important here

File: tools/webpack.config.js (hunk 1 above)
Comment (maxammann) [reply tools/webpack.config.js]: Execute in paralell and cache some stuff for a speedup :P

File: src/components/RichLayout/Footer.js (hunk 2 above)
Comment (Schedulaar) [reply src/components/RichLayout/Footer.js]: ah yeah, right. :D

**Pull request merged**